docs/user/production-considerations/production.asciidoc

```diff
@@ -51,11 +51,17 @@ Settings that must be the same:
 [source,js]
 --------
 xpack.security.encryptionKey //decrypting session information
+xpack.security.authc.providers // authentication providers
```

Comment:
__note:__ I know it wasn't mentioned in https://github.com/elastic/kibana/issues/113928, but would you mind also including `xpack.security.session.*`? Various inconsistencies in the session config also can lead to sporadic logouts (e.g. if one instance is configured with the session lifespan, its cleanup job will remove all sessions created without it).

__nit:__ we still support legacy authentication config, it would be safer to say that entire `authc` config should be in sync (e.g. `xpack.security.authc.saml.*`)
```suggestion
xpack.security.authc.* // authentication providers
```